smaps/src/main/webapp/WEB-INF/logging.properties

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# A default java.util.logging configuration.
+# (All App Engine logging is through java.util.logging by default).
+#
+# To use this configuration, copy it into your application's WEB-INF
```

Comment:
This is in our file as per the directions, so maybe we can delete this part of the comment.

Reply:
Should I delete the whole section from lines 17-26?

Reply:
Yes, I tried to highlight all those lines when I made the comment so that would be clear but it doesn't seem to show that. I need to look into these code reviews/comments more, hopefully there are better ways to do this.